### PR TITLE
finished delete route

### PR DESCRIPTION
**Pull request by Moniarchy**
@jamestewartjr @jrob8577

### CODE REVIEW

**Comment by Moniarchy** [comment]
Merging this but if there are comments from either of you please still leave them and we will make changes on a new branch!

File: routes/books.js [hunk 1]
```diff
@@ -14,6 +14,14 @@ router.post('/new', (req, res, next) => {
     .catch(error => { res.render('error', {error})})
 })
 
+router.get('/delete/:id', (req, res, next) => {
+  const id = req.params.id
+  db.deleteBookById(id)
+  .then(id => res.redirect('/') )
```
Comment (jrobcodes) [inline routes/books.js]: Indentation off

File: schema.sql [hunk 2]
```diff
@@ -3,7 +3,7 @@ CREATE TABLE books(
   id SERIAL PRIMARY KEY,
   title VARCHAR(255),
   author VARCHAR(255),
-  image_url VARCHAR(255),
+  image_url TEXT,
```
Comment (jrobcodes) [inline schema.sql]: Why text?  Seems a little too big for a URL.  Check out this interesting SO post: http://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

File: views/details.pug [hunk 3]
```diff
@@ -8,3 +8,6 @@ block content
       li
         img(src=book.image_url)
       li= book.description
+
+      form(name='delete', id='delete-form', action='/books/delete/'+book.id, method='get')
```
Comment (jrobcodes) [inline views/details.pug]: Are we using ES6?  We could use string interpolation here: `/books/delete/${book.id}`

File: schema.sql (hunk 2 above)
Comment (Moniarchy) [reply schema.sql]: I'm changing it back to VARCHAR after reading your comment and Stack Overflow. The original problem was that all of our image urls were so very long, much longer than 255 characters.

Comment (jrobcodes) [reply schema.sql]: It makes sense to have it longer than 255 - I'd choose a number that was a little more constrained based on the SO post though (2000 maybe?)

**Pull request merged**